quickstarts/microsoft.containerregistry/container-registry-with-policies-and-diagnostics/main.bicep

```diff
@@ -0,0 +1,168 @@
+@minLength(5)
+@maxLength(50)
+@description('Azure Container Registry name')
+param acrName string = 'acr${uniqueString(resourceGroup().id)}'
+
+@description('Azure Container Registry location')
+param location string = resourceGroup().location
+
+@description('Azure Container Registry tier')
+@allowed([
+  'Basic'
+  'Standard'
+  'Premium'
+  'Classic'
+])
+param acrSku string = 'Basic'
+
+@description('Enable System Assign Identity for Azure Container Registry')
+param enableSystemIdentity bool = false
+
+@description('The list of user identity resource ids to associate with the Azure Container Registry')
+param userAssignedIdentities object = {}
```

Comment:
can we add metadata to this param to show the schema of the object that's expected for the param value?

Reply:
done